src/main/java/org/apache/pulsar/manager/interceptor/WebAppConfigurer.java

```diff
@@ -51,6 +51,7 @@ public void addInterceptors(InterceptorRegistry registry) {
                 .excludePathPatterns("/doc.html")
                 // BKVM
                 .excludePathPatterns("/bkvm")
+                .excludePathPatterns("/pulsar-manager/saml/**")
```

Comment:
our endpoint is /sso right? Should this be /pulsar-manager/sso/**

Reply:
our endpoint is /saml/sso @shiv4289 
here /saml is mapped at controller level and **  to exclude all method in that controller from authentication. 
This makes me rethink to change the excludePathPatterns as /pulsar-manager/saml/sso to make only one method accessible without authentication.